--- usbpcap/USBPcapDriver/USBPcap.inx	2025-08-14 17:36:40
+++ Code Patches/USBPcap.inx	2025-08-14 17:55:12
@@ -6,6 +6,7 @@
 Provider            = %PROVIDER%
 CatalogFile.NTx86   = USBPcapx86.cat
 CatalogFile.NTamd64 = USBPcapamd64.cat
+CatalogFile.NTarm64 = USBPcaparm64.cat
 
 [DestinationDirs]
 DefaultDestDir = 12
@@ -32,6 +33,14 @@
 DelFiles  = @USBPcap.sys
 DelReg    = USBPcap.DelReg
 
+[DefaultInstall.NTarm64]
+CopyFiles = @USBPcap.sys
+AddReg    = USBPcap.AddReg
+
+[DefaultUninstall.NTarm64]
+DelFiles  = @USBPcap.sys
+DelReg    = USBPcap.DelReg
+
 [USBPcap.AddReg]
 ;Add USBPcap to UpperFilters
 HKLM, System\CurrentControlSet\Control\Class\{36FC9E60-C465-11CF-8056-444553540000}, UpperFilters, 0x00010008, USBPcap
@@ -52,6 +61,12 @@
 [DefaultUninstall.NTamd64.Services]
 DelService = USBPcap,0x00000200
 
+[DefaultInstall.NTarm64.Services]
+AddService = USBPcap,,Service.AddService
+
+[DefaultUninstall.NTarm64.Services]
+DelService = USBPcap,0x00000200
+
 [Service.AddService]
 DisplayName    = %USBPcap.SvcDesc%
 ServiceType    = 1               ; SERVICE_KERNEL_DRIVER
